### PR TITLE
Update proposal.md with Objective and background.

### PR DESCRIPTION
**Pull request by mRileyMacK**
Objective and background.

### CODE REVIEW

File: proposal.md [hunk 1]
```diff
@@ -17,19 +17,19 @@ Cheryl Schramm
 * [Timetable](#timetable)
 * [Facilities](#facilities)
 * [Degree Relevance](#degree-relevance)
-	* [Thao-Tran Le-Phuong](#thao-tran-le-phuong)
+	* [Thao-Tran Le-Phuong](#thao-tran-le -phuong)
```
Comment (Thao-Tran) [inline proposal.md]: Remove the space between `le` and `-phuong`

File: proposal.md [hunk 2]
```diff
@@ -17,19 +17,19 @@ Cheryl Schramm
...
 
 ## Background
 Real-time collaboration has been implemented as a feature in word processing software, such as Microsoft Word, Google Docs, and 
 Pages. These applications provide a more professional and formal environment for document writing. Our project aims to provide
-a similar environment geared towards social interaction.
+a similar environment geared towards social interaction. The mini-map feature is currently implemented in VS Code as a way to show the user which part of the document they are currently viewing. Our goal is to implement this feature to show where the other users are in the conversation as well as the user's relative position in the conversation. Current messaging applications follow a more rigid messaging format. Messages in a conversation are often presented in cronological order and are uneditable. Our application will strive to break from these conventions to provide a fully editable conversation where the user can type where ever they so choose.
```
Comment (Thao-Tran) [inline proposal.md]: Break the lines up. I usually use the point where the words are wrapped around.

Comment (Thao-Tran) [reply proposal.md]: You spelled `chronological` wrong and `wherever` is one word.

File: proposal.md [hunk 3]
```diff
@@ -17,19 +17,19 @@ Cheryl Schramm
...
-	* [Thao-Tran Le-Phuong](#thao-tran-le-phuong)
+	* [Thao-Tran Le-Phuong](#thao-tran-le -phuong)
 	* [Honor Lopes](#honor-lopes)
 	* [Riley MacKinnon](#riley-mackinnon)
 	* [Igor Veselinovic](#igor-veselinovic)
 
 ## Objectives
 The objective of this project is to develop a messaging platform that is formatted as a blank document that allows for 
-real-time collaboration between users.
+real-time collaboration between users. The platform will display conversations in a document format where users will be able to add or delete anything that they, or others, have said. The platform will also have a 'mini-map' to display where other users' cursors are in the conversation.
```
Comment (Thao-Tran) [inline proposal.md]: Maybe talk about how we want to build upon the current real-time collaboration implementation.

**Review by Thao-Tran — CHANGES_REQUESTED** [review]

**Review by Thao-Tran — APPROVED** [review]
Looks good to me!

**Review by igorveselinovic — APPROVED** [review]
Looks good

**Pull request merged**